### PR TITLE
Landing Page: Walkthrough component

### PR DESCRIPTION
**Pull request by D-Pagey**
[Trello - Building Landing Page: Walkthrough Component](https://trello.com/c/ofsAqOXd)

- Not sure if I should change all styles to Polished?

### CODE REVIEW

File: src/components/HomePage/components/Steps/styles.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    width: 25%;
+
+    &:last-child {
+        border: 0;
+    }
+`;
+
+export const Image = styled.img`
+    height: auto;
+    width: ${(props) => props.width || '100px'};
```
Comment (navgarcha) [inline src/components/HomePage/components/Steps/styles.js]: `width: ${({ width = 100 }) => width};`

File: src/components/HomePage/components/Steps/styles.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    width: 100%;
+`;
+
+export const Wrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    margin: ${spacing.big} auto;
+`;
+
+export const DivStep = styled.div`
```
Comment (navgarcha) [inline src/components/HomePage/components/Steps/styles.js]: just `Step`

File: src/components/HomePage/components/Steps/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import imageCalendar from './assets/image-calendar.png';
+import imageLoot from './assets/image-loot.png';
+import imageMultiChips from './assets/image-3chips.png';
+import imageChip from './assets/image-chip.png';
```
Comment (navgarcha) [inline src/components/HomePage/components/Steps/index.js]: remove the `image-` prefix here

File: src/components/HomePage/components/Steps/index.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import * as S from './styles';
+
+const Steps = () => {
```
Comment (navgarcha) [inline src/components/HomePage/components/Steps/index.js]: not sure about the name of this component...perhaps `Walkthrough` or something more descriptive?

Comment (D-Pagey) [reply src/components/HomePage/components/Steps/index.js]: yeh good point

File: src/components/HomePage/components/Walkthrough/index.test.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import Steps from '.';
+
+describe('Steps component', () => {
```
Comment (navgarcha) [inline src/components/HomePage/components/Walkthrough/index.test.js]: `Walkthrough component`

File: src/components/HomePage/components/Walkthrough/index.test.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import Steps from '.';
```
Comment (navgarcha) [inline src/components/HomePage/components/Walkthrough/index.test.js]: `Walkthrough`

Comment (D-Pagey) [inline src/components/HomePage/components/Walkthrough/index.test.js]: change to walkthrough

**Review by navgarcha — APPROVED** [review]

**Comment by navgarcha** [comment]
Approved pending last changes

File: src/components/HomePage/components/Walkthrough/styles.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+`;
+
+export const Subtitle = styled.h3`
+    font-size: ${fontSize.large};
+    ${margin(spacing.medium, spacing.none)}
+    text-align: center;
+`;
+
+export const Text = styled.p`
+    margin: ${spacing.none};
```
Comment (navgarcha) [inline src/components/HomePage/components/Walkthrough/styles.js]: why do we need 'margin: 0' here? the normalise should handle this?

**Pull request merged**